docs/spec/slashing/begin-block.md

```diff
@@ -75,7 +76,9 @@ This ensures that offending validators are punished the same amount whether they
 act as a single validator with X stake or as N validators with collectively X
 stake.
 
-## Automatic Unbonding
+Double signature slashes are capped by the slashing period as described in [state-machine.md](state-machine.md).
```

Comment:
what do you mean by `slashes are capped by the slashing period` - I thought that "cap" referred to the amount one could be slashed, not by the period in which discovered actions where slashable by. Maybe we could say `Double signature slashes are time bound by` or something like that